gravitee-am-repository/gravitee-am-repository-jdbc-api/src/main/java/io/gravitee/am/repository/jdbc/provider/impl/R2DBCConnectionProvider.java

```diff
@@ -47,8 +50,8 @@
 @org.springframework.context.annotation.Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
 public class R2DBCConnectionProvider implements ConnectionProvider<ConnectionFactory, R2DBCConnectionConfiguration>, InitializingBean, R2DBCSpringBeanAccessor {
 
-    @Value("${oauth2.use-management-settings:true}")
-    private boolean useManagementSettings;
+    private boolean useOAuth2Settings;
```

Comment:
**issue**: looks like useOAuth2Settings is never assigned

Reply:
fixed